packages/web-runtime/src/pages/account.vue

```diff
@@ -134,18 +126,15 @@ export default defineComponent({
       return null
     },
     groupNames() {
-      return this.groups.join(', ')
+      if (this.capabilities.spaces?.enabled) {
+        return this.user.groups.map((group) => group.displayName).join(', ')
```

Comment:
Acceptance tests correctly fail for users without groups. Do this instead:
```suggestion
        return (this.user.groups || []).map((group) => group.displayName).join(', ')
```

Reply:
ah, nice, yours is even better 👍

Reply:
<3